test/api/handlers_test.go

```diff
@@ -78,6 +78,10 @@ func TestHandlers(t *testing.T) {
 		{name: "Challenge-Response/Registration", test: testRegistration},
 		// GET /register, POST /register, GET /login, POST /login
 		{name: "Challenge-Response/Login", test: testLogin},
+		// GET /user/apikey, POST /user/apikey, DELETE /user/apikey/:apiKey
+		{name: "APIKeysFlow", test: testAPIKeysFlow},
+		// POST /user/apikey, GET /user/stats?api_key
```

Comment:
took me a while to figure out what this means 😛

Reply:
The name or the comments?

The comments are a bit wacky. I started them as little notes for myself while writing the tests and then I somehow kept them. I'm not convinced it's a good idea to have them there, as they might not be terribly useful for others. Should I remove them?

Reply:
If you want to keep them I'd drop a quick comment in line 51 to explain what they mean.

Reply:
After thinking about it a bit, I don't think they bring enough value.